CoffeeChat/Controller/EditingViewController.swift

```diff
@@ -82,36 +82,61 @@ class EditingViewController: UIViewController {
     private let tableView = UITableView(frame: .zero, style: .grouped)
 
     // MARK: - Display Settings
-    var showsGroups = true
-    private var showingLess = true
-    private var hideAfter = 3 // Doesn't display more [hideAfter] categories if showingLess is true
+    private var showsGroups = true
+    private var showingLessRows = true
+    private var numberHiddenShownRows = 3 // Displays only [numberHiddenShownRows] if showingLessRows is true
 
     private var sections: [Section] = []
+
     // More Section and count
```

Comment:
This comment doesn't really tell me anything, and the variable name `moreSection` to the outside reader is unclear.